### PR TITLE
✨ Add rule to favicon optimisation

### PR DESCRIPTION
**Pull request by jackreimers**

### CODE REVIEW

File: rules/optimise-your-favicon/rule.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<!--endintro-->
+
+## What is a Favicon?
+
+Favicons are most commonly used to display a custom icon on a browser tab, but they can also be used to alter the display of your website in other cases.
+
+Some places the favicon is used are:
+- Browser Tabs
+- Android and iPhone Icons
+- Windows and Mac Icons
```
Comment (bradystroud) [inline rules/optimise-your-favicon/rule.md:26]: condense this section to a sentence

File: rules/optimise-your-favicon/rule.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+uri: optimise-your-favicon
+created: 2023-07-28T12:00:00.0000000Z
+authors:
+- title: Jack Reimers
+  url: https://ssw.com.au/people/jack-reimers
+related:
+- 
+---
+
+Favicons are commonplace in web development, but did you know your favicon may not be properly optimised for all devices?
```
Comment (bradystroud) [inline rules/optimise-your-favicon/rule.md:15]: I prefer answering the question (rule title) in the intro, and then going into more detail after

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by UlyssesMaclaren — APPROVED** [review]

**Review by Hona — CHANGES_REQUESTED** [review]
screenshots I would 150% zoom & exclude random browser UI

**Review by jaydenalchin — APPROVED** [review]

**Comment by bradystroud** [comment]
The images you want to use haven't been added

**Comment by bradystroud** [comment]
> The images you want to use haven't been uploaded

Fixed

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by bradystroud — APPROVED** [review]

**Review by Hona — APPROVED** [review]

**Pull request merged**